### ISSUE TITLE
did you consider to put that into a file type plugin

### ISSUE THREAD

**Issue by schriftgestalt** [smugpie/amiga-bitmap-font-tools#1]
That way you could doe File > Open or even drag the file onto the dock icon or double click the font file to open it.

**Comment by justsmugpie** [comment]
Hi Georg! Yes, definitely going to have a go at this - time is a bit short at the moment but I will keep you posted!

**Comment by smugpie** [comment]
Hi @schriftgestalt - I'm having a look at this now using code based on the BDF plugin. As an experiment I'm going to try this with JSON files (.afontjson suffix) to start with. However I'm having trouble registering that file type in MacOS.

I can see a "BDFApp.app" in the repo, which has a `UTExportedTypeDeclarations` key in its `info.plist`. Do I need a similar app ("AmigaJSONApp.app") in my repo? And how would I go about setting this up? Or can I get away with adding that key to the `info.plist` in the root of my plugin?

**Comment by schriftgestalt** [comment]
Yes. You need such an app, too. You can basically copy the app, rename it and edit the info plist.

**Comment by smugpie** [comment]
Excellent. I'll try that. Thanks!

**Comment by smugpie** [comment]
It's working! Thanks again @schriftgestalt 🎉
